packages/blade/ios/Podfile.lock

```diff
@@ -466,10 +472,10 @@ SPEC CHECKSUMS:
   RNCAsyncStorage: ea6b5c280997b2b32a587793163b1f10e580c4f7
   RNDateTimePicker: d16386fda012db9e1fac5d2f8281a91254dfee99
   RNGestureHandler: 071d7a9ad81e8b83fe7663b303d132406a7d8f39
-  RNReanimated: f6d11918cc6b85881a5256dcfe68b2cac8832e4d
+  RNReanimated: f066aa91ef8ef7b2a9baed4c15ec06f5e5fb5d11
   RNSVG: 302bfc9905bd8122f08966dc2ce2d07b7b52b9f8
   Yoga: b316a990bb6d115afa1b436b5626ac7c61717d17
 
 PODFILE CHECKSUM: 572e41323005ea0d32663d81a2a36d83084502f2
 
-COCOAPODS: 1.11.3
+COCOAPODS: 1.12.1
```

Comment:
Had to do `pod install` and it changed it to `COCOAPODS: 1.11.3`

Should blade developers upgrade their cocoapods versions?